### DISCUSSION TITLE
Inscription sent with default sat amount

### DISCUSSION THREAD

**Discussion by the-chain-gamer** [ordinals/ord#1515] [Q&A] [answered]
is there anyway to "speed up" that transaction with a higher fee?

**Comment by rgray00** [comment, marked as answer]
not quite RBF  - but for your next inscription, it looks like you can increase the default fee via:

```
ord wallet inscribe --fee-rate 6.0 inscription.png
```

**Comment by casey** [comment]
See @rgray19's comment for next time. The fix for this is https://github.com/casey/ord/issues/1519, allowing fee-bumping of transactions. All transactions already signal for opt-in RBF, but the fee-bumping code hasn't been written.

**Reply by the-chain-gamer** [reply]
I was able to do this with a transaction yesterday using sparrow wallet. today I ran into the same issue but the parent transaction doesn't get sped up. 

https://mempool.space/tx/4c55b80899accb039a6a21985126adf653ea2436de902111f6e59b2ad7dfd21d

This is the transaction - the parent is stuck at 2
